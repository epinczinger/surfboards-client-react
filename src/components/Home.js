import React from 'react';
import { Link } from 'react-router-dom';

const Home = () => (
  <div className="w-fullbg-white font-semibold min-h-screen shadow-md py-5 sm:mx-auto overflow-hidden flex flex-col justify-around">
    <div className="text-3xl text-gray-800 text-center">
      <h1>Waves, waves waves.</h1>
    </div>
    <div className="sm:flex sm:my-2 justify-around">
      <img
        className="himg w-50 object-cover"
        src="https://cdn.pixabay.com/photo/2016/03/09/15/16/wave-1246560_960_720.jpg"
        alt="surfboard"
      />
    </div>
    <div className="text-3xl text-gray-800 text-center">
      <h1>
        Everyone wants to be there, but first you need some
        <Link
          to="/surfboards"
          className="text-blue-500"
        >
          {' '}
          Equipment
        </Link>
      </h1>
    </div>
  </div>
);

export default Home;
