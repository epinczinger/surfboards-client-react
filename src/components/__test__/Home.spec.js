import React from "react";
import renderer from "react-test-renderer";
import Home from "../Home";
import { BrowserRouter as Router } from 'react-router-dom';

describe("The Home component", () => {
  it("renders as expected matching snapshot", () => {
    const tree = renderer
      .create(
        <Router>
          <Home />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
