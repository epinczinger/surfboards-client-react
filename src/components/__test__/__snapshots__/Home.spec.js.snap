// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home component renders as expected matching snapshot 1`] = `
<div
  className="w-fullbg-white font-semibold min-h-screen shadow-md py-5 sm:mx-auto overflow-hidden flex flex-col justify-around"
>
  <div
    className="text-3xl text-gray-800 text-center"
  >
    <h1>
      Waves, waves waves.
    </h1>
  </div>
  <div
    className="sm:flex sm:my-2 justify-around"
  >
    <img
      alt="surfboard"
      className="himg w-50 object-cover"
      src="https://cdn.pixabay.com/photo/2016/03/09/15/16/wave-1246560_960_720.jpg"
    />
  </div>
  <div
    className="text-3xl text-gray-800 text-center"
  >
    <h1>
      Everyone wants to be there, but first you need some
      <a
        className="text-blue-500"
        href="/surfboards"
        onClick={[Function]}
      >
         
        Equipment
      </a>
    </h1>
  </div>
</div>
`;
